inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -146,8 +146,13 @@ def arxiv_plot_extract(obj, eng):
                 return
 
             lb = LiteratureBuilder(source='arxiv', record=obj.data)
+            file_extension_pattern = re.compile('.+(\.[a-zA-Z]+)$')
             for index, plot in enumerate(plots):
-                plot_name = plot.get('name')
+                file_extension = ''
+                match = file_extension_pattern.match(plot.get('url'))
```

Comment:
I don't understand: what puts an extension in the URL in the first place?

Reply:
I was not appending the extension in the popoluated url field. But in the plot there is the url field that has the extension of the file. So I just extract it and append it.